nat-lab/run_local.py

```diff
@@ -5,21 +5,55 @@
 import subprocess
 import sys
 import time
+from datetime import datetime
+from threading import Thread
 from typing import List, Optional, Dict, Any
 
 PROJECT_ROOT = os.path.normpath(os.path.dirname(os.path.realpath(__file__)) + "/../..")
 
 TEST_TIMEOUT = 120
 
 
+def add_timestamps_to_output_lines(input_stream, output_stream):
+    for line in input_stream:
+        line = line.decode("ascii").rstrip()
```

Comment:
Are you sure we will not have utf8 bytes here?

Reply:
To be honest I have no idea, but utf8 is indeed safer. I will change this to utf8.